rotkehlchen/tests/unit/decoders/test_zksync.py

```diff
@@ -104,3 +104,93 @@ def test_zksync_lite_deposit(database, ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert expected_events == events
+
+
+@pytest.mark.vcr
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_zksync_lite_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+    """Test a transaction with the Withdrawal event"""
+    tx_hex = deserialize_evm_tx_hash('0x234407968b9a688be3fb37cf7ff8ef3b4168d6cd85ec45b8344bb2a88832f982')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    timestamp = TimestampMS(1604326368000)
+    user_address = ethereum_accounts[0]
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    eth_str = '1.4437093'
+    dai_str = '1691.92749999'
+    pan_str = '1586.6'
+    usdc_str = '57.25'
+    usdt_str = '15.2'
+    A_PAN = Asset('eip155:1/erc20:0xD56daC73A4d6766464b38ec6D91eB45Ce7457c44')
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_ETH.resolve_to_asset_with_name_and_type(),
+            balance=Balance(amount=FVal(eth_str)),
+            location_label=user_address,
+            notes=f'Withdraw {eth_str} ETH from zksync',
+            counterparty=CPT_ZKSYNC,
+            address=ZKSYNC_BRIDGE,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=176,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_DAI.resolve_to_asset_with_name_and_type(),
+            balance=Balance(amount=FVal(dai_str)),
+            location_label=user_address,
+            notes=f'Withdraw {dai_str} DAI from zksync',
+            counterparty=CPT_ZKSYNC,
+            address=ZKSYNC_BRIDGE,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=177,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_PAN.resolve_to_asset_with_name_and_type(),
+            balance=Balance(amount=FVal(pan_str)),
+            location_label=user_address,
+            notes=f'Withdraw {pan_str} PAN from zksync',
+            counterparty=CPT_ZKSYNC,
+            address=ZKSYNC_BRIDGE,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=178,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=A_USDC.resolve_to_asset_with_name_and_type(),
```

Comment:
there is no need to resolve them since assets are compared using the identifier

```suggestion
            asset=A_USDC,
```

Equivalent change for the places where you did the same